src/LoginPage/LoginForm.jsx

```diff
@@ -1,46 +1,55 @@
 import { memo } from 'react';
 
-export default memo(({ onClick, isLogin }) => ((
-  <form>
-    {isLogin ? null : (
-      <>
-        <input
-          name="first"
-          type="text"
-          placeholder="First Name"
-        />
-        <input
-          name="last"
-          type="text"
-          placeholder="Last Name"
-        />
-      </>
-    )}
-    <input
-      name="email"
-      type="email"
-      placeholder="Email"
-    />
-    <input
-      name="password"
-      type="password"
-      placeholder="Password"
-      autoComplete="off"
-    />
-    {isLogin ? (
-      <button type="submit">Log In</button>)
-      : (<button type="submit">Sign Up</button>)}
+export default memo(({ isLogin, onClick, onChange }) => {
+  function handleChange(event) {
+    const { target: { name, value } } = event;
```

Comment:
```suggestion
function handleChange({ target: { name, value }}) {
```
알고 계실 거 같긴한데~ 저같은 경우엔 이렇게 바로 적어주는게 더 편하긴 하더라구요~ㅎㅎ 이건 참고사항입니다!

Reply:
엇? 저는 생각 못한 방법인데요??? 감사합니다🙏🏼